.github/workflows/ci-cd.yml

```diff
@@ -51,7 +52,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [windows-2019, ubuntu-latest, macos-11]
-        node: [16.x]
+        node: [18.x, 20.x]
```

Comment:
I believe we should keep `16.x` in the matrix (perhaps `[16.x, 18.x]`) to test for backwards compatibility given that builtins and extensions in VS Code will be using `Node 16x` APIs.

If we attempt to build VS Code with a newer Node version we get the [warning](https://github.com/microsoft/vscode/blob/be900a4b816a6cf83f3413f99bf6dcdd447ad7de/build/npm/preinstall.js#L16-L18):

```
vscode ±|main|→ node -v
v18.17.0

vscode ±|main|→ yarn
yarn install v1.22.19
$ node build/npm/preinstall.js
*** Warning: Versions of node.js >= 17 have not been tested.

[1/4] Resolving packages...
[2/4] Fetching packages...
[3/4] Linking dependencies...
...
```

And their CI makes use of Node 16:

https://github.com/microsoft/vscode/blob/be900a4b816a6cf83f3413f99bf6dcdd447ad7de/.github/workflows/ci.yml#L116-L118

Reply:
Done.